terraform-tests/postgresql_flexible_server_test.go

```diff
@@ -0,0 +1,210 @@
+package terraformtests
+
+import (
+	. "csbbrokerpakazure/terraform-tests/helpers"
+	tfjson "github.com/hashicorp/terraform-json"
+	. "github.com/onsi/gomega/gstruct"
+
+	"path"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("PostgreSQL Flexible Server", Label("postgresql-flexible-server"), Ordered, func() {
+	const (
+		instanceName      = "csb-postgresql-flexible-server-instance"
+		resourceGroupName = "csb-resource-group"
+		dbName            = "csb-db"
+	)
+
+	var (
+		plan                  tfjson.Plan
+		terraformProvisionDir string
+	)
+
+	defaultVars := map[string]any{
```

Comment:
The Ginkgo docs explain why this is bad practise: https://onsi.github.io/ginkgo/#extracting-common-setup-beforeeach
(I appreciate it was probably copied from elsewhere)

Reply:
Yep totally agree - have seen this go wrong many times! 
Unfortunately all out terraform test have exactly the same setup but I have changed it for this one to be in the initial `BeforeAll`